### ISSUE TITLE
Inquiry on installation management tool for Naoqi OS

### ISSUE THREAD

**Issue by PhilipAmadasun** [incognite-lab/Pepper-Controller#7]
Not a question directly related to repo. Please be so kind to respond if you have the knowledge. This is an inquiry about the Naoqi OS. Afaik, the OS is based off of Gentoo Linux. Gentoo use **portage (emerge)** for software installation, removal, updates and upgrades correct? The Naoqi OS on my lab's pepper does not have emerge or equo ---`bash: command not found`. I'm at a loss on what the manager could be or if the OS even has a manager installed? If it does not have an installation manager, how will  I be able to do things like:

1. Install python2.7  libraries. pip (version 1.4.1) does not work, it does not seem to be able to access repos, or perhaps its another issue:
``` 
 pip install SpeechRecognition:
Downloading/unpacking SPeechRecognition
  Cannot fetch index base URL https://pypi.python.org/simple/
  Could not find any downloads that satisfy the requirement SPeechRecognition
Cleaning up...
No distributions at all found for SPeechRecognition
Storing complete log in /home/nao/.pip/pip.log 
```

2.  General package upgrades like upgrading pip or python.

**Comment by PhilipAmadasun** [comment]
On further research the pepper manufacturer purposefully disable the installation manager. If your pepper has pip 1.4.1 and/or python 2.7.6, there is  pretty much no way of upgrading them as they are too old for even that.

**Comment by gabinsane** [comment]
Hi, I know what you are talking about. I have been working on another, newly bought Pepper robot with NaoQi 2.5.7 and Python 2.7 and the scenario is the same - the main problem is an outdated openssl on the robot, which prevents from upgrading pip and installing packages. We wrote to the support but so far no reasonable response. 
I managed to manually copy some packages directly on the robot and install them from source via `python setup.py install` - it was possible to do with the SpeechRecognition module for example. But it is of course very inconvenient and there are many other problems.
